app/Http/Transformers/SelectlistTransformer.php

```diff
@@ -33,7 +33,7 @@ public function transformSelectlist (LengthAwarePaginator $select_items)
         }
 
         $results = [
-            'items' => $items_array,
```

Comment:
Is it possible to change this back to `items`? Even though bootstrap tables doesn't use these select lists, it would keep it consistent across the API. We don't really document the select list API endpoints, but one never really knows how folks use the things they root around and find.

Reply:
Yup! No problem.